### ISSUE TITLE
Enhancement suggestion: Include count-down for days as well

### ISSUE THREAD

**Issue by MortenLoberg** [pmongloid/flipdown-timer-card#5]
Hi ! 
I was looking for a pretty count-down card to use for counting down towards an event, and came across your flipdown-timer-card which is very neat and good looking.

I do however miss the inclusion of days in the counter.

My suggestion goes like this:
- Let user choose between Days | Hours | Days+Hours
- If Days, then let the user set a fixed date for an event, which then the timer is counting down towards when started, but showing only number of Days left until the event
- If Hours, then let the user set a fixed number of hours (so this is like it is working now)
- If Days+Hours, then let the user set a fixed date and time for an event, which then the timer is counting down towards when started, and displaying both Days, Hours, Minutes and Seconds left until the event (it should here also be possible to choose not to show the seconds)

Keep up the good work! :-)

**Comment by pmongloid** [comment]
I think too long duration for timer is not so useful because a timer entity doesn't survive ha restart.
Maybe it's better to support input_datetime entity to show remaing time to specific moment.

**Comment by kongjudas** [comment]
> I think too long duration for timer is not so useful because a timer entity doesn't survive ha restart.
> Maybe it's better to support input_datetime entity to show remaing time to specific moment.

Yes, I second that. If it worked both ways, it would be a complete counter. If days are impossible, three or four digit hours would be a nice compromise.

**Comment by HybridRCG** [comment]
Seconds this request. :)

**Comment by CptKalibak** [comment]
> Seconds this request. :)

Yes, i was looking for something to countdown a week or two.

**Comment by pcuser42** [comment]
Adding my support for this feature as well. In my case, I'm looking at using it for a holiday countdown so would be starting months in advance.

**Comment by bluestreak101** [comment]
This card has some decent potential for a few different items. I found it looking for this exact feature. Been keen to support this too
